src/video/damage_tracker.h

```diff
@@ -5,14 +5,21 @@
 #include <stdbool.h>
 
 typedef struct damage_tracker {
-    vga_index dirty_range_start;
-    vga_index dirty_range_end;
+    vga_index dirty_range_first;
+    vga_index dirty_range_last;
     bool dirty;
 } damage_tracker;
 
 void damage_reset(damage_tracker *tracker);
 void damage_copy(damage_tracker *dst, const damage_tracker *src);
 void damage_combine(damage_tracker *dst, const damage_tracker *src);
-void damage_set_range(damage_tracker *tracker, vga_index start, vga_index end);
+
+// Marks a contigous range as damaged.
```

Comment:
For docs, the doxygen syntax we use elsewhere would be nice here, too.

```
/**
 * \brief blah blah
 *
 * \param tracker blah
 * \return blah
 */
 ```

Reply:
Btw -- if it seems that hte common pattern is damage_add_range(tracker, start, start + count); -- maybe something like this could/should be added:

```
damage_add_amount(tracker, start, count);
```

Or somesuch

Reply:
Added doxygen comments to everything in damage_tracker.h. I don't think damage_add_amount is going to be an improvement to clarity, and am not currently feeling creative with names.

Reply:
It's fine, it's not related to this PR anyways.